R/merge.R

```diff
@@ -29,16 +29,22 @@ merge.data.table = function(x, y, by = NULL, by.x = NULL, by.y = NULL, all = FAL
 
   ## set up 'by'/'by.x'/'by.y'
   if ( (!is.null(by.x) || !is.null(by.y)) && length(by.x)!=length(by.y) )
-    stopf("`by.x` and `by.y` must be of same length.")
+    stopf("by.x and by.y must be of same length.")
```

Comment:
why this change? backticks serve to highlight that these are code objects, not plain English terms.

Reply:
Sorry, I see this was requested elsewhere. Let's continue discussion there: https://github.com/Rdatatable/data.table/pull/6713#discussion_r1912605078